google/cloud/functions/internal/compiler_info_test.cc

```diff
@@ -31,7 +34,10 @@ TEST(CompilerInfo, CompilerVersion) {
   auto const cv = CompilerVersion();
   EXPECT_NE(cv, "");
   // Look for something that looks vaguely like an X.Y version number.
-  EXPECT_THAT(cv, ::testing::ContainsRegex(R"([0-9]+.[0-9]+)"));
+  // Cannot use a regex because GTest does not have them with MSVC.
```

Comment:
`Cannot use a regex` is too general given that we're just about to use `std::regex`.  Perhaps mention `ContainsRegex` explicitly.

Reply:
Done.